pages/manifesto.js

```diff
@@ -27,6 +28,12 @@ export default function Manifesto() {
       <div className="my-16">
         <Image src={Manifesto1} alt="" priority></Image>
       </div>
+      <div className="relative aspect-video mt-8 lg:mt-16 shadow-xl rounded-xl">
```

Comment:
The corners do not appear rounded.

Reply:
sorry for the late reply, i don't think they should be rounded anyway 
the page is all edgy 

<img width="1440" alt="Screenshot 2023-11-22 at 10 48 13 AM" src="https://github.com/nation3/website/assets/94828675/1c86e5ed-b6a2-405a-baf1-bf9c7c7d2f91">

i will push an update